src/test/java/uk/nhs/hee/tis/revalidation/connection/service/GmcClientServiceTest.java

```diff
@@ -152,6 +153,25 @@ void shouldHandleEmptySubmissionDateOnTryAddDoctorRequest() {
     assertThat(gmcConnectionResponseDto.getReturnCode(), is(returnCode));
   }
 
+  @Test
+  void shouldHandleNullSubmissionDateOnTryAddDoctorRequest() {
+    when(webServiceTemplate.marshalSendAndReceive(any(String.class), any(
+        TryAddDoctor.class), any(SoapActionCallback.class))).thenReturn(tryAddDoctorResponse);
+    when(tryAddDoctorResponse.getTryAddDoctorResult()).thenReturn(tryAddDoctorResponseCT);
+    when(tryAddDoctorResponseCT.getClientRequestID()).thenReturn(gmcId);
+    when(tryAddDoctorResponseCT.getGMCRequestID()).thenReturn(gmcRequestId);
+    when(tryAddDoctorResponseCT.getSubmissionDate()).thenReturn(null);
```

Comment:
This is the default behaviour when not mocking.  If it's here because you want to be explicit (readability) then that is ok by me.

Reply:
:point_up: 
You can remove `when(tryAddDoctorResponseCT.getSubmissionDate()).thenReturn(null);` unless you specifically want that line to be included.

Reply:
Will give it a tidy